server/src/visitors/YmlVisitorHelper.ts

```diff
@@ -0,0 +1,62 @@
+import { FieldContext } from '../grammar';
+import { connection } from '../server';
+
+const BEGINNING_QUOTES_REGEX = /^("""|")\s*/;
+const ENDING_QUOTES_REGEX = /\s*("""|")$/;
```

Comment:
I had almost forgotten xd

```suggestion

/** Used to trim YML documentation. */
const BEGINNING_QUOTES_REGEX = /^("""|")\s*/;
/** Used to trim YML documentation. */
const ENDING_QUOTES_REGEX = /\s*("""|")$/;
```